code-server/Dockerfile

```
@@ -0,0 +1,7 @@
+FROM lscr.io/linuxserver/code-server:4.103.2 AS code-server-build
+
+RUN echo "hello";
+
+FROM ubuntu:latest AS runtime
+
+COPY --from=code-server-build /app /app
```

```suggestion
COPY --from=lscr.io/linuxserver/code-server:4.103.2 /app /app
```

Might work to do like this, not sure